### PR TITLE
[jsk_fetch_startup] Update charge level warning to prevent battery degradation

### PR DESCRIPTION
**Pull request by tkmtnt7000**
Duplicate of https://github.com/knorth55/jsk_robot/pull/165

This PR updates charge level warning to prevent battery degradation. (40% --> 80%)
The purpose is to prevent the battery from deteriorating due to low battery charge caused by the robot being left unattended.

### CODE REVIEW

**Comment by 708yamaguchi** [comment]
Nice work!

**Review by k-okada — COMMENTED** [review]
@tkmtnt7000 do we still need this, after replace battery to new one? (or did we replace the battery?)

File: jsk_fetch_robot/jsk_fetch_startup/launch/fetch_bringup.launch [hunk 1]
```diff
@@ -22,7 +22,7 @@
         respawn="true" output="screen">
     <rosparam>
       duration: 180
-      charge_level_threshold: 40.0
+      charge_level_threshold: 80.0
```
Comment (tkmtnt7000) [inline jsk_fetch_robot/jsk_fetch_startup/launch/fetch_bringup.launch:25]: I just found this change has already merged in https://github.com/jsk-ros-pkg/jsk_robot/pull/1431 ( d2e919bead1bbfc6b7d3507aa554773b63c8cda9)

**Comment by tkmtnt7000** [comment]
We replaced Fetch batteries with genuine Fetchrobotics ones.

We were just talking among Fetch users about how Fetch's battery threshold of 80% is too frequent to issue warnings.
While it's important to warn at a certain level to prevent rapid battery degradation, it is not be a good idea to set the battery threshold too high for excessive fear of battery degradation in Fetch.
So, for example, we are currently discussing the possibility of setting the shutdown threshold to 40%-50% and the warning threshold to 60%.

**Comment by sktometometo** [comment]
This feature is merged in another PR. So closed.

**Pull request closed without merging**